### PR TITLE
task/DES-2898: Active job status nav styling

### PR DESCRIPTION
**Pull request by rstijerina**
## Overview: ##
Set background color for Job Status Nav when hovered and when selected.

## PR Status: ##

* [X] Ready.

## Related Jira tickets: ##

* [DES-2898](https://tacc-main.atlassian.net/browse/DES-2898)

## Testing Steps: ##
1. Ensure job status nav appears as below when hovered, and selected. Confirm no link cursor when active and hovered while in Job Status.

## UI Photos:

Screenshot 2024-06-17 at 6 30 58 PM

### CODE REVIEW

**Review by sophia-massie — APPROVED** [review]
LGTM!

File: client/modules/workspace/src/JobStatusNav/JobStatusNav.module.css [hunk 1]
```diff
@@ -6,3 +6,16 @@
 .badge {
   margin: 0 10px;
 }
+
+.root:hover,
+.root:focus,
+.highlighted-row,
+.highlighted-row:hover,
+.highlighted-row:focus {
```
Comment (wesleyboar) [inline client/modules/workspace/src/JobStatusNav/JobStatusNav.module.css:14]: This code is good. I merely offer new syntax in context.

```suggestion
root:is(:hover, :focus),
.highlighted-row:is(:default, :hover, :focus) {
```

I have used [`:is(…)`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) enough to know this works. The [`:default`](https://developer.mozilla.org/en-US/docs/Web/CSS/:default) is new to me, but widely available.

File: client/src/styles.css [hunk 2]
```diff
@@ -132,7 +132,7 @@
 }
 
 /* DO scroll app category menu */
-.ant-layout-sider-children > header + div + ul {
+.ant-layout-sider-children > a + div + ul {
```
Comment (wesleyboar) [inline client/src/styles.css:135]: Are you sure no `.ant-layout-sider-children > header + div + ul` use cases still exist?

**Review by wesleyboar — COMMENTED** [review]
Thoughts.

File: client/modules/workspace/src/JobStatusNav/JobStatusNav.module.css (hunk 1 above)
Comment (wesleyboar) [reply client/modules/workspace/src/JobStatusNav/JobStatusNav.module.css:14]: I removed proposed use of [`:default`](https://developer.mozilla.org/en-US/docs/Web/CSS/:default), which is stated to be only applicable to form elements. The [:is(…)](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) suggestion is valid.

File: client/src/styles.css (hunk 2 above)
Comment (rstijerina) [reply client/src/styles.css:135]: Fairly certain

**Pull request merged**